src/adapt.ts

```diff
@@ -452,15 +448,15 @@ export function adaptOutputUtil(
     for (const outputSpec of outputSpecs) {
         const currentValue = result[resultIndex];
         if (COMMON_NUMERIC_TYPES.includes(outputSpec.type)) {
-            adapted[outputNameOrDefault(outputSpec.name)] = currentValue;
+            adapted[outputSpec.name] = currentValue;
```

Comment:
Don't we risk `outputSpec.name` to be undefined? If it's undefined, is it necessarily the only property? We need to have a cairo function in our tests which has an unnamed return parameter.

Reply:
So, from my investigation,
* In case of cairo 0
   - `outputSpec.name` wouldn't be `undefined` because any external function's return type must be named.

* In case of cairo 1
   - `outputSpec.name` could be `undefined` if it's the only property, `contract1.cairo` on our example repo has `get_balance` func with an unnamed return type.